changes.md

```diff
@@ -0,0 +1,18 @@
+# PostgreSQL installer release 17.6-1 (2025-08-14)
+## Changes 🛠️
+- **Dependencies (macOS & Windows)**
+    - Update `pgAdmin4` to version `9.6`.
+    - Update `libxml2` to version `2.14.5`.
+    - Update `curl` to version `8.15.0`.
+    - Update `openssl` to version `3.0.17`.
+
+- **Additional dependencies (macOS)**
+    - Update `libpng2` to version `1.6.50`.
+    - Update `e2fsprogs` to version `1.47.3`.
+
+
+## Bug Fixes 🐛
+- **Fix:** Improvements in DoCmd function in initcluster script (Issue #347)
+- **Fix:** disable wrapInScript for getlocales.ps1 as it was failing in few conditions (Issue #343)
```

Comment:
remove this line

Reply:
and add a bug fix for https://github.com/EnterpriseDB/edb-installers/commit/f5f5c74f15735adbdb4cabb0c9b50c7f148f84d3